plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -73,21 +79,21 @@ protected TestTable createTableWithDefaultColumns()
     }
 
     @Override
-    public void testCreateSchema()
+    public void testAddColumn()
     {
-        // schema creation is not supported
+        throw new SkipException("Adding columns via SQL are not supported until adding columns with comments are supported");
```

Comment:
The comment was misleading. ALTER TABLE ADD COLUMN doesn't depend on column comments at all.

Maybe simply say `Adding columns are not supported by the connector`.